rotkehlchen/chain/substrate/manager.py

```diff
@@ -388,7 +388,7 @@ def _get_chain_properties(
         try:
             chain_properties = SubstrateChainProperties(
                 ss58_format=properties['ss58Format'],
-                token=Asset(properties['tokenSymbol']),
+                token=CryptoAsset(properties['tokenSymbol']),
```

Comment:
As I also wrote in the previous review, shouldn't we have handling of a an unexpected/unknown asset here?

Reply:
hmmm no this is actually already caught properly from before. No action needed here.